moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js

```diff
@@ -73,13 +73,34 @@ const useColumns = ({
           return user ? `${user.first_name} ${user.last_name}` : "";
         },
         renderEditCell: (props) => {
+          // the team member object for the current row
+          const currentRowMember =
+            data?.moped_project_by_pk?.moped_proj_personnel.filter(
+              (user) => user.project_personnel_id === props.id
+            );
+          // the existing team members on this project
+          const existingTeamMembers =
+            data?.moped_project_by_pk?.moped_proj_personnel.map(
+              (option) => option.moped_user.email
+            );
+          // filter out existing team members from list of options unless they are the current row member
+          // that way the current member remains an option when editing a row
```

Comment:
this feels like it could be moved up in scope and wrapped with `useMemo`. Similar to `useWorkgroupLookup` in this same file.

I don't think there will be long lists of team members in projects so performance is debatable, but I think it would be an easy win to move up and pass into `useColumns` since it is used by all rows.

also, what do you think about sticking to using user IDs to find objects in this new code? I might be missing a limitation that makes it necessary to use email in some places.

Reply:
i updated to move `existingTeamMembers` up in scope and pass to `useColumns`, ty for that rec! also no reason to use email i think i picked that bc it was easier to debug with and then never went back to reevaluate my choice, `user_id` is definitely the better option here!

Reply:
cool cool thank you for updating 🙏